plugin/build_baron.go

```diff
@@ -178,6 +178,29 @@ func BbGetConfig(settings *evergreen.Settings) map[string]evergreen.BuildBaronPr
 }
 
 func BbGetProject(settings *evergreen.Settings, projectId string) (evergreen.BuildBaronProject, bool) {
+	flags, err := evergreen.GetServiceFlags()
+	if err != nil {
+		return evergreen.BuildBaronProject{}, false
+	}
+	if flags.PluginAdminPageDisabled {
+		lastGoodVersion, err := model.FindVersionByLastKnownGoodConfig(projectId, -1)
+		if err != nil || lastGoodVersion == nil {
+			return evergreen.BuildBaronProject{}, false
+		}
+		parserProject, err := model.ParserProjectFindOneById(lastGoodVersion.Id)
+		if err != nil {
+			return evergreen.BuildBaronProject{}, false
```

Comment:
Can we add a comment to explain what's happening / the bool return value, now that this function is more complex?